hydrus/helpers.py

```diff
@@ -530,3 +530,21 @@ def parse_collection_members(object_: dict) -> dict:
             return error_response(error)
     object_['members'] = members
     return object_
+
+
+def get_fragments(resource: str) -> dict:
+    """Gets a fragment of the main hydra vocabulary.
+
+    :param resource: Fragment specified in the request parameters
+    :type resource: str
+    :return: Object referred to by the fragment
+    :rtype: dict
+    """
+    resourcedict = dict()
+    gen_doc = get_doc().generate()
+    resourcedict['@context'] = {i: gen_doc['@context'][i] for i in sorted(gen_doc['@context'])}
```

Comment:
```
resourcedict['@context'] = {i: gen_doc['@context'][i] for i in sorted(gen_doc['@context'])}
```
Is there any particular reason we need to sort here(`sorted`)?

Reply:
So the context which was being returned, was jumbled and I think I had noticed sorted keys in contexts before. Is there no specification for sorting?? Then I can remove the sorted keyword

Reply:
afaik, I don't think there is any specification for the keys to be in any sorted order. You can remove the `sorted` keyword.

/cc @priyanshunayan @chrizandr